ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/SketchOnMap/SketchOnMap.qml

```diff
@@ -52,10 +57,27 @@ Rectangle {
         GraphicsOverlay {
             id: sketchOverlay
         }
+    }
+
+    // Display an option to delete the selected vertex if the user right-clicks or taps and holds when the sketch editor is started
+    MouseArea {
+        anchors.fill: parent
+        acceptedButtons: {
+            Qt.RightButton
+            Qt.TapAndHoldGesture
```

Comment:
did this end up working? I remember having issues with it, and we discussed adding a delete selected vertex button

Reply:
@ldanzinger `Qt.TapAndHoldGesture` unfortunately still doesn't work as expected. Also I added a delete selected vertex button.